### ISSUE TITLE
Test-PSCodeBlock not recognized

### ISSUE THREAD

**Issue by pkunze** [dfinke/ConvertFromMarkdown#6]
I Installed miktext and pandoc (1.19.2), but I am unable to convert from markdown.

```ps
ConvertFrom-Markdown -markdownFile .\changelog.md -OutputType PDF
```

results in 

> Die Benennung "Test-PSCodeBlock" wurde nicht als Name eines Cmdlet, einer Funktion, einer Skriptdatei oder eines ausführbaren Programms
> erkannt. Überprüfen Sie die Schreibweise des Namens, oder ob der Pfad korrekt ist (sofern enthalten), und wiederholen Sie den Vorgang.
> In C:\Program Files\WindowsPowerShell\Modules\ConvertFromMarkdown\1.1.0\ConvertFromMarkdown.psm1:17 Zeichen:5
> \+     Test-PSCodeBlock  -markdownFile $markdownFile -outputPath $output ...
> \+     ~~~~~~~~~~~~~~~~
>         \+ CategoryInfo          : ObjectNotFound: (Test-PSCodeBlock:String) [], CommandNotFoundException
>         \+ FullyQualifiedErrorId : CommandNotFoundException
> 

The Module seems to be installed correctly though..

![image](https://user-images.githubusercontent.com/1416798/48072511-f9838180-e1dc-11e8-9cad-21e8e778db79.png)

**Comment by dfinke** [comment]
Could you also post the `changelog.md`?

**Comment by pkunze** [comment]
Hey, thanks for the quick response! Content of `changelog.md` below:

```md
# Changelog

## unreleased

- Test3
- Test4

## 4.0.30

- Test1
- Test2

```

**Comment by pkunze** [comment]
Btw. the error above roughly translates to 

> The Identifier "Test-PSCodeBlock" was not recognized as the name of a Cmdlet, a function, a scriptfile or an executable program.
> Check the spelling of the Idetifier and if the path is correct (if contained) and repeat.
> In C:\Program Files\WindowsPowerShell\Modules\ConvertFromMarkdown\1.1.0\ConvertFromMarkdown.psm1:17 Column:5
> \+ Test-PSCodeBlock -markdownFile $markdownFile -outputPath $output ...
> \+ ~~~~~~~~~~~~~~~~
> \+ CategoryInfo : ObjectNotFound: (Test-PSCodeBlock:String) [], CommandNotFoundException
> \+ FullyQualifiedErrorId : CommandNotFoundException

**Comment by dfinke** [comment]
Thanks for the translation, I had to head off to Google translate.

`ConvertFrom-Markdown` works by using  `<!-- CHAPTER START --> and <!-- CHAPTER END -->` in your markdown.

https://github.com/dfinke/ConvertFromMarkdown/blob/master/README.md#chapters

On my side the `ConvertFrom-Markdown -markdownFile .\changelog.md -OutputType PDF` just hangs and produces nothing. I'll look at adding a check and message of those are not found.

Try adding those start and end "tags" and see if you still get an error.

**Comment by pkunze** [comment]
yeah, you are right, it generates the pdf now. however, the error message still pops up (even though it seems to work)?

**Comment by ranabanerjee15** [comment]
Hi Doug,
first of all, really love your work. for ImportExcel and ConvertToMarkdown
KEEP IT UP

Now I get this following error(below)
* I have added  and  as advised above into the markdown. still no luck.
* I get the same error when I attempt to convert it to HTML
help would be much appriciated.

ERROR DETAILS:

Test-PSCodeBlock : The term 'Test-PSCodeBlock' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\banerjeeR\OneDrive - Queensland Health\Documents\WindowsPowerShell\Modules\ConvertFromMarkdown\1.1.0\ConvertFromMarkdown.psm1:17 char:5
+     Test-PSCodeBlock  -markdownFile $markdownFile -outputPath $output ...
+     ~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Test-PSCodeBlock:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
 
pandoc : --smart/-S has been removed.  Use +smart or -smart extension instead.
At line:1 char:1
+ pandoc C:\Users\banerjeeR\OneDrive - Queensland Health\Documents\Migr ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (--smart/-S has ...ension instead.:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
Fo
r example
:
 p
and
oc
 
-f ma
rkd
own+
s
m
a
rt
 -t
 m
a
r
k
d
ow
n
-
sm
art
.
Try
 
pa
n
d
oc
.e
xe
 -
-he
lp
 fo
r 
m
or
e 
i
nf
orm
a
t
io
n.
Invoke-Item : Cannot find path 'C:\Users\banerjeeR\OneDrive - Queensland Health\Documents\MigrationOps\QHO365MigrationOps\Docs\manuscript\book.Docx' because it does not exist.
At C:\Users\banerjeeR\OneDrive - Queensland Health\Documents\WindowsPowerShell\Modules\ConvertFromMarkdown\1.1.0\ConvertFromMarkdown.psm1:42 char:13
+             Invoke-Item $outFile
+             ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\banerj...cript\book.Docx:String) [Invoke-Item], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.InvokeItemCommand

**Comment by dfinke** [comment]
Thanks! Is the md file you're working with on OneDrive? Could you move it to a different location add try to process it?

Otherwise, could you post it and I'll try to look at it.

**Comment by dfinke** [comment]
Thanks! Is the md file you're working with on OneDrive? Could you move it to a different location add try to process it?

Otherwise, could you post it and I'll try to look at it.

**Comment by ranabanerjee15** [comment]
Hello Doug,

I have tried moving the md files out of the one drive
[Validate-QHMigrationUsers.txt](https://github.com/dfinke/ConvertFromMarkdown/files/2853716/Validate-QHMigrationUsers.txt)
Please change the extension to .md
this seems to be an error with any md file i attempt
 but still get his issue.

**Comment by dfinke** [comment]
Quick scan looks like you need to add at least one `<!-- CHAPTER START -->` and `<!-- CHAPTER END -->`

https://github.com/dfinke/ConvertFromMarkdown#chapters

**Comment by Agazoth** [comment]
I have the same issue, but running a forced import of the module showed:
```ps
 Import-Module convertfrommarkdown -force
. : The script 'TestPSCodeBlock.ps1' cannot be run because the following modules that are specified by the "#requires"
statements of the script are missing: PSScriptAnalyzer.
At C:\Program Files\WindowsPowerShell\Modules\convertfrommarkdown\1.1.0\ConvertFromMarkdown.psm1:2 char:3
+ . $PSScriptRoot\TestPSCodeBlock.ps1
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (TestPSCodeBlock.ps1:String) [], ScriptRequiresException
    + FullyQualifiedErrorId : ScriptRequiresMissingModules
```

When dot-sourcing modules in the psm1 file you wont catch the #requires

After running:
```ps
Install-Module PSScriptAnalyzer
```

everything runs smooth as butter.

Thanks for a great module!

**Comment by imunisasi** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Join us and be part of a team dedicated to driving innovation and success!

## How to Apply:
To apply for this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.eric496, @SAIHTOOMYAT, @ismaelsadeeq, @mlesk, @stefan-vatov, @draveness, @sjimi, @feDann, @salsowelim, @miguelmtzinf, @veryordinally, @dexhunter, @jack5432109, @R0nni333, @Edwin-kagiri, @hbakhtiyor, @ztid, @sibozhang, @ludope, @thiagosouza

**Comment by dfinke** [comment]
Thanks for the update @Agazoth
